policy/modules/system/init.te

```diff
@@ -445,6 +448,7 @@ ifdef(`init_systemd',`
 	# mount-setup
 	fs_unmount_autofs(init_t)
 	fs_getattr_pstore_dirs(init_t)
+	fs_create_pstore_dirs(init_t)
```

Comment:
I wonder what this is all about:

```
[root@dssp4test2 dssp5]# mkdir /sys/fs/pstore/test
mkdir: cannot create directory ‘/sys/fs/pstore/test’: Operation not permitted
```

Reply:
Honestly not sure either, but it comes from this denial during boot:
```
audit: type=1400 audit(1616864322.098:3): avc:  denied  { create } for  pid=1 comm="systemd" name="pstore" scontext=system_u:system_r:init_t:s0 tcontext=system_u:object_r:pstore_t:s0 tclass=dir permissive=1
```
Also to note that this is after I have updated my dracut configuration such as to not include the selinux or udev-rules modules.

Reply:
After looking at this again I found that in my case, I can still boot in enforcing without this access.

Reply:
You should copy fedora for systemd or dracut when fixing selinux denials

Reply:
I've seen this denial on a yocto-based system, but it has been a while. I assume it's valid for allow. It may be some syscall or other kernel interface that is triggering a dir creation in pstore. I haven't dug into it.

Reply:
OK. I can allow the access again.